ibis/backends/exasol/__init__.py

```diff
@@ -61,13 +61,31 @@ def do_connect(
         engine = sa.create_engine(url, poolclass=sa.pool.StaticPool)
         super().do_connect(engine)
 
+    @property
+    def inspector(self):
+        with warnings.catch_warnings():
+            warnings.filterwarnings("ignore", category=sa.exc.RemovedIn20Warning)
+            return super().inspector
+
     @contextmanager
     def begin(self):
         with warnings.catch_warnings():
             warnings.filterwarnings("ignore", category=sa.exc.RemovedIn20Warning)
             with super().begin() as con:
                 yield con
 
+    def list_tables(self, like=None, database=None):
+        with warnings.catch_warnings():
+            warnings.filterwarnings("ignore", category=sa.exc.RemovedIn20Warning)
+            return super().list_tables(like=like, database=database)
+
+    def _get_sqla_table(
+        self, name: str, schema: str | None = None, autoload: bool = True, **kwargs: Any
+    ) -> sa.Table:
+        with warnings.catch_warnings():
+            warnings.filterwarnings("ignore", category=sa.exc.RemovedIn20Warning)
+            return super()._get_sqla_table(name, schema, autoload, **kwargs)
+
```

Comment:
Hm, this should be a pass through I think. The reason to keep this is to allow the exasol dialect, which appears to be making some sqlalchemy 1.x-only calls to do so without ibis tests failing (we enable the failure to ensure ibis can work with both sqlalchemy 1.x and 2.x, since some dialects support both).

What happens if you `export SQLALCHEMY_WARN_20=1` and then run the relevant tests?

Reply:
Long term the exasol dialect should probably be compatible with both major versions of sqlalchemy, many of the 1.x calls are calling `execute` with a `str`, which can be changed to `exec_driver_sql` to support 1.x and 2.x.

Reply:
My assumption is, it is due to the signature mismatch. Just wanted to get your feedback on it. Also was wondering if I missed something else. The info regarding SQLA 1.x vs 2.x was already quite valuable to me. :+1: 

`BaseAlchemyBackend`
```python
    def _get_sqla_table(
        self,
        name: str,
        *,
        namespace: ops.Namespace = ops.Namespace(),  # noqa: B008
        autoload: bool = True,
        **_: Any,
    ) -> sa.Table:

```

`ExasolBackend`
```python
    def _get_sqla_table(
        self,
        name: str,
        schema: str | None = None,
        autoload: bool = True,
        **kwargs: Any
    ) -> sa.Table:
```

Reply:
Alright
```python
    def _get_sqla_table(
            self, name: str,
            namespace: ops.Namespace = ops.Namespace(),  # noqa: B008
            autoload: bool = True, **kwargs: Any
    ) -> sa.Table:
        with warnings.catch_warnings():
            warnings.filterwarnings("ignore", category=sa.exc.RemovedIn20Warning)
            return super()._get_sqla_table(name=name, namespace=namespace, autoload=autoload, **kwargs)
```

did the trick ;).